examples/wasm-shell/components/wasm-terminal.tsx

```diff
@@ -94,16 +50,17 @@ const fetchCommandHandler = async (
   }
 
   if (!didInitWasmTransformer) {
-    await wasmInit(wasmTransformerWasmUrl);
     didInitWasmTransformer = true;
   }
 
-  return lowerI64Imports(wasmBinary);
+  return await lowerI64Imports(wasmBinary);
 };
 
 /**
  * A simple preact wrapper around the Wasm Terminal
  */
+const processWorkerUrl = document.getElementById("worker").src;
```

Comment:
So you are no longer using the optimized version, and now you are using the unoptimized version of the bundle. The process worker should be loaded by the unopmitized bundle automatically, and shouldn't require you to offer a URL for the user?

Do you get a `console.warn` warning you of this? If not, I need to file an issue for me to fix 😄

Reply:
Yeah, this was due to some issues with parcel and the logic it does when you import `.wasm` files. This solution is more like a hack that works :)

> Do you get a console.warn warning you of this? If not, I need to file an issue for me to fix 😄

Nope, I just tried and doesn't emit a warning.

Reply:
Ahhh I see why: https://github.com/wasmerio/wasmer-js/blob/master/packages/wasm-terminal/lib/wasm-terminal-config.ts#L41

Since Rollup isn't handling the bundle, this never gets properly added. 

So that's fine, glad that it works still 😄